modules/connatixBidAdapter.js

```diff
@@ -225,13 +248,15 @@ export const spec = {
    */
   buildRequests: (validBidRequests = [], bidderRequest = {}) => {
     const bidRequests = _getBidRequests(validBidRequests);
+    let cnxIds = readFromAllStorages(CNX_IDS) || CNX_IDS_VALUES;
```

Comment:
I'd try catch reading from local storage just in case

Reply:
in readFromAllStorages i checked if the values are undefined to make sure i dont get any errors, so i dont think a try catch is really necessary, unless there's another way it can return an error?